frontend/src/lib/components/PropertyFilters/components/PropertyFilterButton.tsx

```diff
@@ -19,44 +21,63 @@ export interface PropertyFilterButtonProps {
     onClose?: () => void
     children?: string
     item: AnyPropertyFilter
-    style?: React.CSSProperties
 }
 
-export const PropertyFilterButton = React.forwardRef<HTMLElement, PropertyFilterButtonProps>(
```

Comment:
Looks like the popover is no longer positioned properly
<img width="965" alt="Screenshot 2023-12-08 at 16 01 15" src="https://github.com/PostHog/posthog/assets/4550621/d6e13ba6-535a-491e-9714-674a7c2b9fb1">

Reply:
Glad you found this. Re-added the ref and it works as before now.